tests/bindings/python/test_printer.py

```diff
@@ -17,7 +17,7 @@ def test_print_model(self):
 
         # std::string printModel(ModelPtr model)
         p = Printer()
-        self.assertIsInstance(p.printModel(Model()), str)
+        self.assertIsInstance(p.printModel(Model(), False), str)
```

Comment:
The second parameter (`False`) is optional in C++ and it should also be in Python.

Reply:
cca111e7